index.html

```diff
@@ -68,6 +67,123 @@ <h2 id="support-text">I'm a Software Developer</h2>
     <img id="main-img" src="assets/illustration.svg" alt="">
   </section>
 
+  <section id="recent-works">
```

Comment:
- [ ] As required by the project requirements, the main container of the projects should have a `display:grid`

Reply:
Hi @topeogunleye , i just checked and there is no part in the activity where it specifies that main should be display:grid.
There is only one part that says that the works section should use both grid and flex which I did in my project.
Could you confirm where did you get that main must be grid? thank you